### PR TITLE
main/modemmanager: add service file

### PR DESCRIPTION
**Pull request by alkaline-system**
still learning to write services, complain if something isn't right

### CODE REVIEW

**Comment by NovaAndrom3da** [comment]
Can you squash the commits please

**Comment by nekopsykose** [comment]
i think by default git pull will create merge commits like that, so you need `git pull --rebase` or similar

File: main/modemmanager/files/modemmanager [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# modemmanager service
+
+type            = process
+command         = /usr/bin/modemmanager
+before          = network.target
+depends-on      = dbus
+depends-on      = pre-network.target
+restart         = true
+smooth-recovery = true
```
Comment (nekopsykose) [inline main/modemmanager/files/modemmanager:9]:
```suggestion
type = process
command = /usr/bin/modemmanager
before = network.target
depends-on = dbus
depends-on = pre-network.target
smooth-recovery = true
```

don't need indents or restart=true anymore (yeah should update the other files eventually..)

File: main/modemmanager/template.py [hunk 2]
```diff
@@ -47,6 +47,10 @@
 sha256 = "6c8f8720737a3788e394c700f36236278c9de09d76069a079e6f1daaf08b2768"
 
 
+def post_install(self):
+    self.install_service(self.files_path / "modemmanager", enable=True)
```
Comment (nekopsykose) [inline main/modemmanager/template.py:51]:
```suggestion
    self.install_service(self.files_path / "modemmanager")
```

shouldn't be enabled by default on install

**Comment by alkaline-system** [comment]
redid the commit (with suggested changes)
we good now

**Comment by alkaline-system** [comment]
no, we're good NOW
sorry lol

File: main/modemmanager/template.py (hunk 2 above)
Comment (nekopsykose) [reply main/modemmanager/template.py:51]: still have to remove this

File: main/modemmanager/files/modemmanager [hunk 3]
```diff
@@ -0,0 +1,6 @@
+type = process
+command = /usr/bin/modemmanager
```
Comment (nekopsykose) [inline main/modemmanager/files/modemmanager:2]:
```suggestion
command = /usr/bin/ModemManager
```

the lowercase version doesn't exist

File: main/modemmanager/template.py (hunk 2 above)
Comment (alkaline-system) [reply main/modemmanager/template.py]: oops

File: main/modemmanager/files/modemmanager [hunk 4]
```diff
@@ -0,0 +1,6 @@
+type = process
```
Comment (triallax) [inline main/modemmanager/files/modemmanager:1]: service files usually have a comment at the beginning

```suggestion
# modemmanager service

type = process
```

Comment (nekopsykose) [reply main/modemmanager/files/modemmanager:1]: tbh they're all just `# {filename} service`, not really super insightful

could probably just start getting rid of those too

Comment (nekopsykose) [reply main/modemmanager/files/modemmanager:1]: will just think about that later

Comment (triallax) [reply main/modemmanager/files/modemmanager:1]: sure

**Pull request merged**